Resource.js

```diff
@@ -282,6 +285,11 @@ class Resource {
       ) {
         return new Date(value);
       }
+
+      if (param.instance === 'Date' && parsedValue) {
+        return new Date(parsedValue);
```

Comment:
Hi @YuryRybakSoftensity,

This seems to break queries with formats like `http://localhost:3001/form/5a98653751c0700d00363e04/submission?sort=-modified&modified__lte=2018-03-03T20:54:23`.  getValue is returning on line 290 a new Date from parsedValue which just contains the year, in this case 2018.  Before change documents were being returned, now they're not.

Reply:
Thanks @douglaselee. We are looking at it right now.

Reply:
Happy to help @travist.  In my case I simply appended 'Z' to the date/times so they were treated as ISO.

Reply:
Thanks. @YuryRybakSoftensity please take a look at ^